### ISSUE TITLE
RSS feed not importing latest content

### ISSUE THREAD

**Issue by nadia847** [MantaRayMedia/alnap#359]
RSS feed: https://www.alnap.org/jobs/rss.xml
View: https://www.alnap.org/admin/structure/views/view/rss_feed/edit/feed_1
Validation: https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fwww.alnap.org%2Fjobs%2Frss.xml (even when namespaces have been remover from the views template)
Views template: /var/www/alnap-live/sites/all/themes/alnap/templates/views/views-view-rss.tpl.php

**Comment by meldera** [comment]
@nadia Got the feed working. Over to you to check/map the fields you need.

**Comment by meldera** [comment]
@nadia see http://alnap-dev.mrmdev2.co.uk/admin/structure/views/view/rss_feed/edit/feed_1 for an example and use "Global: PHP" to construct your final output. Note "Content: Post date (>= -70 days)" is set for testing only...

**Comment by nadia847** [comment]
@Maria-alnap the jobs RSS feed is now pulling in organisation logos. over to you for testing :o)

**Comment by Maria-alnap** [comment]
@nadia847 thanks! I just sent a test to my outlook  it's still not showing up on my end, though (see attached).
![logo in jobs email not showing](https://user-images.githubusercontent.com/29306539/37402053-024ba010-2782-11e8-8b5a-015caf3f5d9d.PNG)

**Comment by meldera** [comment]
@Maria-alnap Could you forward the email to arseni@mantaraymedia.co.uk please?

**Comment by nadia847** [comment]
@Maria-alnap the logos are appearing for me in the preview of your mailchimp and a test i sent out from our mailchimp. Is this working OK for you?

**Comment by Maria-alnap** [comment]
@nadia847 just checked and they're indeed loading now but they look a bit pixelated. Any idea why this might be?

**Comment by Maria-alnap** [comment]
@nadia847  Just got a test of the jobs email and the logos now look huge! I assume you're seeing the same and still tweaking this?

**Comment by nadia847** [comment]
@Maria-alnap the template was pulling in smaller thumbnails which I've now made bigger. I've also created a  test rss campaign in your mailchimp account to send tests to you and myself so I could ensure this was working ok; I'm getting different results from the template in our manta mailchimp than yours so i wanted to make sure your template works.

As mentioned, it's a paused clone of your RSS campaign with Dave's test list as the recipient list so this won;t go out to anyone. If you're happy with the test I just sent, feel free to delete the "TEST Jobs and opportunities updates CLONE" campaign.
